File: src/main/java/no/digdir/informasjonsforvaltning/fdk_dataset_harvester/Application.java
package no.digdir.informasjonsforvaltning.fdk_dataset_harvester;

import no.digdir.informasjonsforvaltning.fdk_dataset_harvester.configuration.ApplicationProperties;
import no.digdir.informasjonsforvaltning.fdk_dataset_harvester.configuration.FusekiProperties;
import org.apache.jena.riot.RIOT;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties({ ApplicationProperties.class, FusekiProperties.class })
public class Application {

    public static void main(String[] args) {
        RIOT.init();
        SpringApplication.run(Application.class, args);
    }

}

File: src/main/kotlin/no/digdir/informasjonsforvaltning/fdk_dataset_harvester/Application.kt
package no.digdir.informasjonsforvaltning.fdk_dataset_harvester

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.SpringApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity

@SpringBootApplication
@ConfigurationPropertiesScan
@EnableWebSecurity
open class Application

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}
